scalding-beam/src/main/scala/com/twitter/scalding/beam_backend/BeamOp.scala

```diff
@@ -34,6 +42,25 @@ sealed abstract class BeamOp[+A] {
 }
 
 object BeamOp extends Serializable {
+  def planMapGroup[K, V, U](
+    pcoll: PCollection[KV[K, java.lang.Iterable[V]]],
+    reduceFn: (K, Iterator[V]) => Iterator[U])(implicit ordK: Ordering[K], kryoCoder: KryoCoder): PCollection[KV[K, java.lang.Iterable[U]]] = {
```

Comment:
I think it's better to keep `coder` as an argument around, and don't make it concrete to `KryoCoder` anywhere